api/docs/scatter_gather_emulation.dox

```diff
@@ -61,7 +61,7 @@ be complete.
 
 
 ```
-vpscatterdd {%k1} %xmm10 -> %rax(,%xmm11,4)[4byte] %k1
+{%raw %}vpscatterdd {%k1} %xmm10 -> %rax(,%xmm11,4)[4byte] %k1{%endraw %}
```

Comment:
nit: All our Liquid tags have a space between the `%` and the tag name.  The same style with a space is what they use in all the Liquid docs I've seen.

Reply:
Added the space.